Traceback (most recent call last):
  File "/home/vkeller/.local/pipx/venvs/jupyter-book/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 154, in wrapped
    asyncio.get_running_loop()
RuntimeError: no running event loop

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/vkeller/.local/pipx/venvs/jupyter-book/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/vkeller/.local/pipx/venvs/jupyter-book/lib/python3.11/site-packages/nbclient/client.py", line 1319, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/vkeller/.local/pipx/venvs/jupyter-book/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 158, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/vkeller/.local/pipx/venvs/jupyter-book/lib/python3.11/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/home/vkeller/.local/pipx/venvs/jupyter-book/lib/python3.11/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/vkeller/.local/pipx/venvs/jupyter-book/lib/python3.11/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
def quizz_info():
    r = input("Qui est le meilleur prof: \n 1 : M. Keller \n 2: un autre ... \n Votre rÃ©ponse:")
    if r == '1':
        print("Bonne rÃ©ponse")
    else:
        print("Mauvaise rÃ©ponse")

def quizz_philo():
    r = input("quizz philo")
    

print("Quizz simple")
print("Jouons ? C'est parti...")
reponse = ''
while reponse != 'q':
    reponse = input("Choisissez votre question: \n 1: informatique \n 2: philo \n Votre rÃ©ponse ('q' pour quitter):")
    if (reponse == '1'):
        quizz_info()
    elif (reponse == '2'):
        quizz_philo()
    elif (reponse == 'q'):
        print("Vous avez dÃ©cidÃ© de quitter")
    else:
        print(reponse,"invalide. Tapez 1 ou 2 ou q pour quitter")

------------------

----- stdout -----
Quizz simple
Jouons ? C'est parti...
------------------

[31m---------------------------------------------------------------------------[39m
[31mStdinNotImplementedError[39m                  Traceback (most recent call last)
[36mCell[39m[36m [39m[32mIn[2][39m[32m, line 16[39m
[32m     14[39m reponse = [33m'[39m[33m'[39m
[32m     15[39m [38;5;28;01mwhile[39;00m reponse != [33m'[39m[33mq[39m[33m'[39m:
[32m---> [39m[32m16[39m     reponse = [38;5;28;43minput[39;49m[43m([49m[33;43m"[39;49m[33;43mChoisissez votre question: [39;49m[38;5;130;43;01m\n[39;49;00m[33;43m 1: informatique [39;49m[38;5;130;43;01m\n[39;49;00m[33;43m 2: philo [39;49m[38;5;130;43;01m\n[39;49;00m[33;43m Votre rÃ©ponse ([39;49m[33;43m'[39;49m[33;43mq[39;49m[33;43m'[39;49m[33;43m pour quitter):[39;49m[33;43m"[39;49m[43m)[49m
[32m     17[39m     [38;5;28;01mif[39;00m (reponse == [33m'[39m[33m1[39m[33m'[39m):
[32m     18[39m         quizz_info()

[36mFile [39m[32m~/.local/pipx/venvs/jupyter-book/lib/python3.11/site-packages/ipykernel/kernelbase.py:1274[39m, in [36mKernel.raw_input[39m[34m(self, prompt)[39m
[32m   1272[39m [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m [38;5;28mself[39m._allow_stdin:
[32m   1273[39m     msg = [33m"[39m[33mraw_input was called, but this frontend does not support input requests.[39m[33m"[39m
[32m-> [39m[32m1274[39m     [38;5;28;01mraise[39;00m StdinNotImplementedError(msg)
[32m   1275[39m [38;5;28;01mreturn[39;00m [38;5;28mself[39m._input_request(
[32m   1276[39m     [38;5;28mstr[39m(prompt),
[32m   1277[39m     [38;5;28mself[39m._parent_ident[[33m"[39m[33mshell[39m[33m"[39m],
[32m   1278[39m     [38;5;28mself[39m.get_parent([33m"[39m[33mshell[39m[33m"[39m),
[32m   1279[39m     password=[38;5;28;01mFalse[39;00m,
[32m   1280[39m )

[31mStdinNotImplementedError[39m: raw_input was called, but this frontend does not support input requests.

